packages/rhsplib/lib/binding.ts

```diff
@@ -21,10 +21,7 @@ import {
 } from "@rev-robotics/rev-hub-core";
 import { SerialParity } from "@rev-robotics/rev-hub-core";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = dirname(__filename);
-const require = createRequire(import.meta.url);
-const addon = require("node-gyp-build")(path.join(__dirname, ".."));
+const addon: any = {};
```

Comment:
This needs to be undone when we move changes back. We did this because we couldn't get it to work with rollup. It makes the ExpansionHubs unusable